pubspec.lock

```diff
@@ -7,42 +7,42 @@ packages:
       name: async
```

Comment:
Add the pubspec.lock file only when you have changed the pubspec.yaml file from your side. In any other case, don't add it to the commit.
What to do now? Revert this back to the original pubspec.lock and then commit.